### ISSUE TITLE
Terminal fails to install (?) 

### ISSUE THREAD

**Issue by KB7GP** [platformio/platformio-atom-ide-terminal#559]
Red window pop up … button "View Issue" pressed and sends me to this page which means nothing to me and no indication as to how I can resolve my issue. I have uninstalled and reinstalled Atom 4 times now with the same Red error screen (you know what they say about doing the same thing over and over expecting a different result). So, Here is what is on my clipboard about the error this may make sense to someone at a higher pay grade than myself (hopefully someone will provide me with a solution?):

The error was thrown from the platformio-ide-terminal package. This issue has already been reported.
View Issue

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.28.0 x64
**Electron**: 2.0.3
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [platformio-ide-terminal](https://github.com/platformio/platformio-atom-ide-terminal) package 2.8.2

### Stack Trace

Failed to activate the platformio-ide-terminal package

```
At Cannot find module 'underscore'

Error: Cannot find module 'underscore'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (~/AppData/Local/atom/app-1.28.0/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.28.0/resources/app/src/module-cache.js:354:58)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/terminal-plus/node_modules/underscore-plus/lib/underscore-plus.js:5:7)
    at /packages/terminal-plus/node_modules/underscore-plus/lib/underscore-plus.js:522:4)
    at /packages/terminal-plus/node_modules/underscore-plus/lib/underscore-plus.js:524:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.28.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/terminal-plus/node_modules/space-pen/lib/space-pen.js:8:9)
    at /packages/terminal-plus/node_modules/space-pen/lib/space-pen.js:634:4)
    at /packages/terminal-plus/node_modules/space-pen/lib/space-pen.js:636:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.28.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/terminal-plus/node_modules/atom-space-pen-views/lib/main.js:4:10)
    at /packages/terminal-plus/node_modules/atom-space-pen-views/lib/main.js:22:4)
    at /packages/terminal-plus/node_modules/atom-space-pen-views/lib/main.js:24:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.28.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.28.0/resources/app/src/compile-cache.js:240:29)
```

### Commands

```
```

### Non-Core Packages

```
atom-terminal 0.8.0 
platformio-ide-terminal 2.8.2 
```

**Comment by the-j0k3r** [comment]
Thank you for your bug report.

You want platformio-ide-terminal 2.8.3 already released and compatible with Atom 1.28.0/electro 2.0.3

Your 2.8.2 will not work with electron 2.0.3 or Atom 1.28.0 so trying to install it is futile.

Also atom-terminal is obsolete.

**Comment by the-j0k3r** [comment]
So I take it you installed it since youre asking https://github.com/platformio/platformio-atom-ide-terminal/issues/560, if so, please close this. Thanks.
